.github/workflows/deploy.yml

```diff
@@ -0,0 +1,27 @@
+# Workflow for deploying app to Firebase
```

Comment:
For an overview on GitHub actions, see [this article](https://docs.github.com/en/actions/getting-started-with-github-actions/overview)